org.jacoco.core/src/org/jacoco/core/instr/Instrumenter.java

```diff
@@ -76,6 +78,15 @@ public void setRemoveSignatures(final boolean flag) {
 	 * 
 	 */
 	public byte[] instrument(final ClassReader reader) {
+		return instrument(reader.b);
+	}
+
+	private byte[] instrument(final byte[] source) {
```

Comment:
Move the private method below the public `instrument()` methods?

Reply:
@marchof To me seems logical that this method is at the beginning, being the main part of process of instrumentation. Also didn't know that we have convention that `public`s should be before `private`s, this is already not respected - private method `instrumentError` is before public method `instrumentAll`. And most importantly - movement will affect diff in a way that will be harder to understand what was changed. If strictly needed (?) - will prefer to do this reordering afterwards separately.

Reply:
@Godin The private method is now called from the public method above and from another public `instrument()` method below. Without the history the placement in the middle looks a bit random to me. But not strictly needed as all.